containers/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9.16-slim-buster
+
+ARG VERSION
+
+WORKDIR /app
+
+RUN pip3 install requests>=2.24 pydantic==1.10.12 jinja2==3.1.2 networkx==3.1 tenacity==8.2.3
```

Comment:
I don't believe that this is needed. I think whatever pyatlan needs will be installed when you install it.

Reply:
Aight 👍